<?xml version="1.0" encoding="UTF-8"?>
<!--This does not replace URDF, and is not an extension of URDF.
    This is a format for representing semantic information about the robot structure.
    A URDF file must exist for this robot as well, where the joints and the links that are referenced are defined
-->
<robot name="humanoid_mani">
    <group_state name="waypoint_offset">
        <joint name="rotate_torso" value="0.35814"/>
        <joint name="rotate_1"     value="1.47026"/>
        <joint name="rotate_3"     value="-0.13508"/>
        <joint name="rotate_5"     value="-0.50893"/>
        <joint name="rotate_tilt"  value="0.0"/>
    </group_state>
    <group_state name="target_offset">
        <joint name="rotate_torso" value="0.35814"/>
        <joint name="rotate_1"     value="1.23464"/>
        <joint name="rotate_3"     value="-0.13508"/>
        <joint name="rotate_5"     value="-0.12252"/>
        <joint name="rotate_tilt"  value="0.0"/>
    </group_state>
    <!--DISABLE COLLISIONS: By default it is assumed that any link of the robot could potentially come into collision with any other link in the robot. This tag disables collision checking between a specified pair of links. -->
    <disable_collisions link1="Head_1" link2="base_link" reason="Never"/>
    <disable_collisions link1="Head_1" link2="ee_link_1" reason="Never"/>
    <disable_collisions link1="Head_1" link2="eye_1" reason="Adjacent"/>
    <disable_collisions link1="Head_1" link2="shoulder_1" reason="Never"/>
    <disable_collisions link1="Head_1" link2="torso_1" reason="Adjacent"/>
    <disable_collisions link1="base_link" link2="ee_link_1" reason="Never"/>
    <disable_collisions link1="base_link" link2="eye_1" reason="Never"/>
    <disable_collisions link1="base_link" link2="shoulder_1" reason="Never"/>
    <disable_collisions link1="base_link" link2="torso_1" reason="Adjacent"/>
    <disable_collisions link1="ee_link_1" link2="eye_1" reason="Never"/>
    <disable_collisions link1="ee_link_1" link2="forearm_1" reason="Adjacent"/>
    <disable_collisions link1="ee_link_1" link2="shoulder_1" reason="Never"/>
    <disable_collisions link1="ee_link_1" link2="upperarm_1" reason="Never"/>
    <disable_collisions link1="eye_1" link2="shoulder_1" reason="Never"/>
    <disable_collisions link1="eye_1" link2="torso_1" reason="Never"/>
    <disable_collisions link1="eye_1" link2="upperarm_1" reason="Never"/>
    <disable_collisions link1="forearm_1" link2="shoulder_1" reason="Never"/>
    <disable_collisions link1="forearm_1" link2="upperarm_1" reason="Adjacent"/>
    <disable_collisions link1="shoulder_1" link2="torso_1" reason="Adjacent"/>
    <disable_collisions link1="shoulder_1" link2="upperarm_1" reason="Adjacent"/>
</robot>
